$NetBSD: patch-CMakeLists.txt,v 1.1 2024/09/04 17:12:00 gdt Exp $

Remove warning that upstream adds w/o checking acceptability.

Reported upstream by email 20240904.
Upstream has addressed for 9.5.x:
  https://github.com/OSGeo/PROJ/commit/de8118616a668fe92f2c0aadc847449b0d0d122a

--- CMakeLists.txt.orig	2024-09-04 16:49:11.007622091 +0000
+++ CMakeLists.txt
@@ -65,7 +65,6 @@ if("${CMAKE_C_COMPILER_ID}" STREQUAL "GN
   )
   set(PROJ_CXX_WARN_FLAGS ${PROJ_common_WARN_FLAGS}
     -Weffc++
-    -Wextra-semi
     # -Wold-style-cast
     -Woverloaded-virtual
     -Wzero-as-null-pointer-constant
@@ -85,7 +84,6 @@ elseif("${CMAKE_C_COMPILER_ID}" MATCHES 
   )
   set(PROJ_CXX_WARN_FLAGS ${PROJ_common_WARN_FLAGS}
     -Weffc++
-    -Wextra-semi
     # -Wold-style-cast
     -Woverloaded-virtual
     -Wshorten-64-to-32
